src/views/PrivacyNotice.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <section>
+    <h1 class="title">Personvern</h1>
+    <h3>Hvilken data samler vi inn?</h3>
+    <!-- TODO: Hvilke data vi (og tredjeparter) samler inn -->
+    <h3>Hvordan samler vi inn data?</h3>
+    <!-- TODO: Hvordan vi (og tredjeparter) samler inn dataene -->
+    <h3>Hvordan bruker vi din data?</h3>
+    Dine data brukes ikke til annet enn det som er strengt nødvendig for å tilby
+    tjenesten.
+    <!-- TODO: Hvordan bruker tredjeparter disse dataene? -->
+    <h3>Hvordan lagrer vi dine data?</h3>
+    <!-- TODO: Hvordan vi (og tredjeparter) lagrer disse dataene -->
+    <h3>Dine rettigheter</h3>
+    Du kan selv slette brukeren din og alle relaterte data på
+    <a href="https://karantenehjelpen.no/my-page">Min side</a>.
```

Comment:
Antar at det er en mer hensiktsmessig måte å gjøre dette på via routeren, om noen vet hvordan; rop ut.

Reply:
`this.$router.push("/my-page")` tror jeg.

Reply:
Eller kanskje uten `this` siden det er i template. Er ingen racer på vue.